src/Command/CreateCommand.php

```diff
@@ -16,10 +16,12 @@ public function configure(): void
              ->addArgument('name', InputArgument::REQUIRED, 'Migration name');
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output): void
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $customName = $input->getArgument('name');
 
         $this->createEmptyMigration($output, $customName);
+
+        return 0;
```

Comment:
I think it should return [ExitCode::OK](https://github.com/yiisoft/yii-console/blob/master/src/ExitCode.php#L32)